docker/vol/secret-service/config.toml

```diff
@@ -2,9 +2,9 @@ seed = "/app/seed"
 network = "regtest"
 
 [tls]
-# key = "/app/tls/key.der"
-# cert = "/app/tls/cert.der"
-# ca = "/app/tls/bridge.ca.der"
+key = "/app/tls/key.der"
+cert = "/app/tls/cert.der"
+ca = "/app/tls/bridge.ca.der"
 
 [transport]
-addr = "127.0.0.1:3000"
+addr = "0.0.0.0:69"
```

Comment:
Why are we using "well-known" ports here?
See: https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers

Reply:
Yes but some users might not have privileges to run that port range.
Do 6969 if you want funny number but that does not have issues.
See the wikipedia entry for well-known ports.